src/main/java/sirius/biz/scripting/mongo/MongoCustomScriptController.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.scripting.mongo;
+
+import sirius.biz.scripting.CustomEventRegistry;
+import sirius.biz.scripting.ScriptingController;
+import sirius.biz.web.BizController;
+import sirius.biz.web.MongoPageHelper;
+import sirius.db.mixing.query.QueryField;
+import sirius.db.mongo.Mango;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.tokenizer.Position;
+import sirius.pasta.noodle.compiler.CompilationContext;
+import sirius.pasta.noodle.compiler.NoodleCompiler;
+import sirius.pasta.noodle.compiler.SourceCodeInfo;
+import sirius.pasta.noodle.sandbox.SandboxMode;
+import sirius.pasta.tagliatelle.compiler.TemplateCompiler;
+import sirius.web.controller.Routed;
+import sirius.web.http.WebContext;
+import sirius.web.security.Permission;
+import sirius.web.services.InternalService;
+import sirius.web.services.JSONStructuredOutput;
+
+/**
+ * Provides the management UI for {@link MongoCustomScript custom scripts}.
+ */
+@Register(framework = MongoCustomEventDispatcherRepository.FRAMEWORK_SCRIPTING_MONGO)
+public class MongoCustomScriptController extends BizController {
+
+    private static final String PARAM_SCRIPT = "script";
+
+    @Part
+    private Mango mango;
+
+    /**
+     * Lists all scripts available for the current tenant.
+     *
+     * @param webContext the request to handle
+     */
+    @Routed("/scripting/scripts")
```

Comment:
**Question:** Are these included in the menu-bar somewhere?

Reply:
done